mono_repo/lib/src/commands/travis.dart

```diff
@@ -312,6 +323,18 @@ esac
 ''';
 }
 
+String _safeEcho(bool prettyAnsi, AnsiCode code, String value) {
+  // Safely escape everything:
```

Comment:
why a block comment instead of a Doc comment?

Reply:
ack